packages/amplify-codegen/tests/commands/models.test.js

```diff
@@ -0,0 +1,113 @@
+const generateModels = require('../../src/commands/models');
+const mock = require('mock-fs');
```

Comment:
It is better to change the importing name to `mockFs`(or others) to avoid confusion